### PR TITLE
API keys

### PR DESCRIPTION
**Pull request by fenik17**
A new way to access the API.

### CODE REVIEW

File: docs/backend-api/how-to/get-api-key.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+{
+  "success": true,
+  "hash": "882fb333405d006df0d5a3f410115e92"
+}
+```                                                             
+
+Where resulting user's session hash is `882fb333405d006df0d5a3f410115e92` (just an example, you will get a different hex string).
+
+After that, you need to get a [list of API keys](../resources/commons/api-keys.md#list) or create a new via using 
+the [/api/key/create](../resources/commons/api-keys.md#create) call using obtained user's session hash:
```
Comment (Astarsys) [inline docs/backend-api/how-to/get-api-key.md:37]: After that, you need to get a [list of API keys](../resources/commons/api-keys.md#list) or create a new one using 
the [/api/key/create](../resources/commons/api-keys.md#create) call with obtained user's session hash:

File: docs/backend-api/how-to/index.md [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+---
+
+# How to
+
+Working with API might seem hard at first, but the goal of our documentation is to assist you in this process and make
+it more approachable.
+
+Our "How to" section has step-by-step examples of working with the Navixy API.
+
+From initial step of obtaining a API key hash to more complicated operations like retrieving a list of devices, tracks or
```
Comment (Astarsys) [inline docs/backend-api/how-to/index.md:13]: an API key hash

File: docs/backend-api/how-to/get-api-key.md (hunk 1 above)
Comment (fenik17) [reply docs/backend-api/how-to/get-api-key.md]: fixed

**Comment by fenik17** [comment]
@Astarsys thanks!

**Pull request merged**